src/app/tabs/registrar/operations.js

```diff
@@ -214,7 +217,13 @@ export const checkIfAlreadyCommitted = domain => async (dispatch) => {
   }
 };
 
-export const revealCommit = domain => async (dispatch) => {
+export const revealCommit = domain => async (dispatch, getState) => {
+  const { rifCost: cost } = getState().registrar;
+
+  const errorMessage = getLanguageString('not_enough_rif_balance');
```

Comment:
is there a way to avoid the 'not_enough_rif_balance' magic value? if there is no easy way is ok.

Reply:
Jummm `not_enough_rif_balance` is the key on the languages json files, I don't see other way of doing this.

Reply:
Finally I find a way to load the corresponding `errorMessage` without needing to use `getLanguageString` function.